lib/challenge.js

```diff
@@ -35,11 +35,17 @@ function prompt1() {
  */
 
 function prompt2() {
-  // YOUR CODE HERE
+  // for (let i = 0; i < 99; i++) {
+  //   if (i % 2 === 0) {
+  //     console.log(i);
+  //   } else {
+  //   }
+  // }
 }
```

Comment:
The empty `else` statement here is not necessary.  I would only include that if there is something specific you want it to do in that case.